icons/kaios.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KaiOS icon</title><path d="M4.47 0a2.81 2.81 0 0 0-2.84 2.78v18.43A2.8 2.8 0 0 0 4.47 24a2.8 2.8 0 0 0 2.85-2.78V2.78A2.83 2.83 0 0 0 4.47 0zm9.4 8.54a2.8 2.8 0 0 0-3.89-.66 2.68 2.68 0 0 0-.68 3.8l8 11.15a2.81 2.81 0 0 0 3.88.66 2.68 2.68 0 0 0 .74-3.72l-.06-.09-7.98-11.14zm3.65-1.18a3.71 3.71 0 0 0 3.74-3.67A3.71 3.71 0 0 0 17.52.02a3.71 3.71 0 0 0-3.75 3.67 3.71 3.71 0 0 0 3.75 3.67z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KaiOS icon</title><path d="M4.4688.0002a2.81 2.81 0 00-2.84 2.7799v18.4295a2.8 2.8 0 002.84 2.79 2.8 2.8 0 002.85-2.78V2.7801A2.83 2.83 0 004.4687 0zm9.3998 8.5397a2.8 2.8 0 00-3.89-.66 2.68 2.68 0 00-.6799 3.8l7.9998 11.1497a2.81 2.81 0 003.8799.66 2.68 2.68 0 00.74-3.72l-.06-.09L13.8786 8.54zm3.6499-1.18a3.71 3.71 0 003.7399-3.6698 3.71 3.71 0 00-3.74-3.67 3.71 3.71 0 00-3.7498 3.67A3.71 3.71 0 0017.5185 7.36z"/></svg>
```

Comment:
Just rebuilt from source, and looks like there's some differences between PR (red) and source (black).
Any chance you could look to rebuild, and potentially reduce to 3 decimals?
![image](https://user-images.githubusercontent.com/1830867/102926655-4e51e700-448d-11eb-834a-f0e524a1228f.png)

Reply:
I've changed the source now to https://www.kaiostech.com/company/press-room. Probably the Dropbox source is pretty outdated.

Then I've rebuilt the from the `KaiOS-Logo-Color-Positive.svg` file in the press kit. Please have another look at it. 😊 

Note: A precision of 4 was needed. I couldn't make it work with a precision of 3.